bin/jobsub_submit

```diff
@@ -67,7 +67,7 @@ def render_files(
     """use jinja to render the templates from srcdir into the dest directory
     using values dict for substitutions
     """
-    if values["debug"] > 0:
+    if values["debug"]:
```

Comment:
Shouldn't this be checking values["verbose"] now?

Reply:
Yes - you're right.  Because if `--debug` is set, that implies `--verbose 1`, but the reverse is not true.  I'll change that.

Reply:
Alright - these are fixed.  Can you please check to make sure it looks good to you now?